packs/ac-feats/Vicious_Rend_PG5ndHwkPEyjJ70k.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "Vicious Rend",
+  "type": "feat",
+  "effects": [],
+  "system": {
+    "description": {
+      "gm": "",
+      "value": "<p>Make two claw, fist, pincer, or talon @UUID[Compendium.pf2e.actionspf2e.VjxZFuUXrCU94MWR]{Strikes} against the same target, applying the multiple attack penalty as normal. If both Strikes hit, the target takes an additional @Damage[1d6[bleed, persistent]]{1d6 persistent bleed} damage. This damage increases to @Damage[2d6[bleed, persistent]]{2d6} if your companion is specialized.</p>"
```

```suggestion
      "value": "<p>Make two claw, fist, pincer, or talon Strikes against the same target, applying the multiple attack penalty as normal. If both Strikes hit, the target takes an additional @Damage[1d6[bleed]] damage. This damage increases to 2d6 if your companion is specialized.</p>"
```
We can DamageAlteration the inline damage too.